css/app.css

```diff
@@ -1,3 +1,820 @@
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-4.png);
+}
+
+.latest-articles-item:nth-child(5) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-5.png);
+}
+
+.latest-articles-item:nth-child(6) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-6.jpg);
+}
+
+.latest-articles-item:nth-child(7) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-7.gif);
+}
+
+.latest-articles-item:nth-child(8) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-8.png);
+}
+
+.latest-articles-item:nth-child(9) .main-article-showcase-header {
+  background:
+    linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.44021358543417366) 29%,
+      rgba(0, 0, 0, 0.5634628851540616) 68%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url(../img/latest-articles-images/latest-articles-image-9.jpg);
+}
+
+.footer-social-icons li:not(:first-child),
+.footer-links li:not(:first-child) {
+  margin-left: 0.75rem;
+}
+
+/* media queries  */
+
+/* 1024 */
+@media (max-width: 1024px) {
+  html {
+    font-size: 80%;
+  }
+
+  .nav-logo {
+    display: flex;
+  }
+
+  main {
+    padding: 40px 20px;
+  }
+
+  .latest-news-grid-container,
+  .latest-funding-grid-container,
+  .latest-deals-grid-container {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .latest-funding-grid-container article:last-child {
+    display: none;
+  }
+
+  .latest-articles {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .main-article-showcase-header {
+    height: 22.5rem;
+  }
+
+  .showcase-item-header {
+    display: none;
+  }
+
+  .showcase-item-main {
+    justify-content: flex-end;
+  }
+}
+
+@media (max-width: 768px) {
+  .nav-main-right {
+    margin-left: auto;
+  }
+
+  .nav-main-left,
+  .nav-top {
+    display: none !important;
+  }
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_